### DISCUSSION TITLE
ICD10 2023

### DISCUSSION THREAD

**Discussion by linyid** [StefanoTrv/simple_icd_10_CM#3] [Ideas]
Hi,

First of all thank you for this wonderful library. The functionality is exactly what I need for my work and I couldn't find any other packages that satisfied my demand.

I was wondering if you could update your data with the newest version of ICD10CM code from 2023? I believe the link to it is:

https://www.cdc.gov/nchs/icd/Comprehensive-Listing-of-ICD-10-CM-Files.htm

Thanks a lot!

**Comment by StefanoTrv** [comment]
Hi!
I'm glad that this library is useful for you. Thank you for letting me know that there's a new version.
I can't update the library immediately because I'm very busy right now. I should be able to update it in the next few weeks, but there's the problem that I need to check that the data from the CDC is correct, since there were some errors in the previous version (see [this commit](https://github.com/StefanoTrv/simple_icd_10_CM/commit/51322047793025be5dd7a27a3f13528e6ac84801)).

**Comment by linyid** [comment]
Hi,

Thank you for the reply! I didn't know that the data from CMS is more complete than that of CDC! Great catch. I'm looking forward to your future updates!

Cheers
